_migration-assistant/getting-started-data-migration.md

```diff
@@ -2,18 +2,20 @@
 layout: default
 title: Getting started with data migration
 nav_order: 10
```

Comment:
The children of this page are not displaying because the parent does not have `has_children: true` specified.

Reply:
There aren't any child pages for this section.

Reply:
There were child pages for "Deploying Migration Assistant" that was not displaying, however. I fixed that in my latest commit.

Reply:
You're right - it was for the "Deploying Migration Assistant" topic, but they are still not displaying.